pkg/services/notification/resource_control/address_list.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2019 The OpenPitrix Authors. All rights reserved.
+// Use of this source code is governed by a Apache license
+// that can be found in the LICENSE file.
+package resource_control
+
+import (
+	"context"
+	"time"
+
+	"openpitrix.io/logger"
+	nfdb "openpitrix.io/notification/pkg/db"
+	"openpitrix.io/notification/pkg/global"
+	"openpitrix.io/notification/pkg/models"
+	"openpitrix.io/notification/pkg/pb"
+	"openpitrix.io/notification/pkg/util/pbutil"
+	"openpitrix.io/notification/pkg/util/stringutil"
+)
+
+func CreateAddressListWithAddrIDs(ctx context.Context, addrList *models.AddressList, addrIds []string) (string, error) {
+	db := global.GetInstance().GetDB()
+	tx := db.Begin()
+	err := tx.Create(&addrList).Error
```

Comment:
Will the tx be useful? tx not used in UpdateAddressAddrLsIdByIds.

Reply:
update.